tools/psa/build_tfm.py

```diff
@@ -419,6 +420,41 @@ def _copy_binaries(source, destination, toolchain, target):
                                             relpath(output_dir, ROOT)))
             shutil.copy2(tfm_secure_bin, output_dir)
 
+def _copy_tfm_ns_files(source):
+    """
+    Copy TF-M NS API files into Mbed OS
+    :param source: Source directory containing TF-M NS API files
+    """
+    with open(join(dirname(__file__), "tfm_ns_import.json")) as ns_import:
+        json_data = json.load(ns_import)
+        data_files = json_data["files"]
+        data_folders = json_data["folders"]
+        logger.info("Copying NS API source from TF-M to Mbed OS")
+        for f in data_files:
+            src_file = join(source, f["src_file"])
+            dst_file = join(ROOT, f["dst_file"])
+            if not isdir(dirname(dst_file)):
+                os.makedirs(dirname(dst_file))
+            try:
+                shutil.copy2(src_file, dst_file)
+            except FileNotFoundError:
+                # Workaround: TF-M build process exports all NS API files to
+                # cmake build folder. The json file `tfm_ns_import.json` contains
+                # list of files and folder relative to cmake build folder. The
+                # only exception is the OS abstraction layer app/os_wrapper_cmsis_rtos_v2.c
+                # which is not exported by TF-M. Therefore it is handled via an
+                # exception
```

Comment:
I recommend adding a full stop to the end of complete sentences. I'd also recommend replacing "via" with "as" in this instance, as well as another comma for readability.

```diff
- Therefore it is handled via an exception
+ Therefore, it is handled as an exception.
```

Reply:
@devran01 Do you want to update this, we can run CI

Reply:
Fixed